### PR TITLE
adding parser and metadata creator

### PR DESCRIPTION
**Pull request by tbeukhof**

### CODE REVIEW

File: gradle/codenarc.groovy [hunk 1]
```diff
@@ -0,0 +1,427 @@
+/*
+ * Copyright 2014 the original author or authors.
```
Comment (aaronzirbes) [inline gradle/codenarc.groovy]: don't commit this

File: epo-install-packages.gradle [hunk 2]
```diff
@@ -8,13 +8,39 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.peoplenet:gradle-plugins:0.1.7'
+        classpath 'com.peoplenet:gradle-plugins:0.5.0'
```
Comment (aaronzirbes) [inline epo-install-packages.gradle]: you can go up to `0.7.0` now

File: gradle/codenarcTest.groovy [hunk 3]
```diff
@@ -0,0 +1,427 @@
+/*
+ * Copyright 2014 the original author or authors.
```
Comment (aaronzirbes) [inline gradle/codenarcTest.groovy]: don't commit this file either

File: playbook.yml [hunk 4]
```diff
@@ -0,0 +1,21 @@
+---
+- hosts: all
+  sudo: yes
+  gather_facts: yes
+  pre_tasks:
+    - name: Create rabbit-adapter-conf staging directory
+      file: path=/pfm/staging state=directory
+    - name: Copy over rabbit-adapter-oembroker config
+      template: src=roles/rabbit-adapter-conf/files/rabbit-adapter-oembroker.conf dest=/pfm/staging
```
Comment (aaronzirbes) [inline playbook.yml]: still installing to `/pfm`???

File: gradle/codenarc.groovy (hunk 1 above)
Comment (tbeukhof) [inline gradle/codenarc.groovy]: Ahh, they keep coming back. I guess I have to add it to the .gitignore.

File: epo-install-packages.gradle (hunk 2 above)
Comment (tbeukhof) [inline epo-install-packages.gradle]: Thanks

File: playbook.yml (hunk 4 above)
Comment (tbeukhof) [inline playbook.yml]: This is a temp location that ends up getting deleted that I was just doing the same as other event processors, but probably can go under /usr/local or /tmp.

Comment (tbeukhof) [inline playbook.yml]: In the refactorings you did yesterday, did any of the projects have rabbit adapters that I can model after? I may not need all this pre_tasks stuff if only needing one rabbit adapter and can simply call it rabbit-adapter.

Comment (tbeukhof) [inline playbook.yml]: Just kidding, this project will need a kafka adapter, not a rabbit adapter.

**Pull request merged**